docs/src/testing/gas-and-resource-estimation.md

```diff
@@ -0,0 +1,65 @@
+# Gas and VM Resources Estimation
+
+`snforge` supports gas and other VM resources estimation for each individual test case. 
+
+It does not calculate the final transaction fee, for details on how fees are calculated, 
+please refer to fee mechanism in [Starknet documentation](https://docs.starknet.io/documentation/architecture_and_concepts/Network_Architecture/fee-mechanism).
```

Comment:
[nit] I think it is more confusing/overwhelming than helpful, but that's my personal feeling